doc/conf.py

```diff
@@ -114,7 +109,10 @@
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-html_theme_options = {}
+html_theme_options = {
+  "github_url": "https://github.com/jupyterhub/binderhub",
+  "twitter_url": "https://twitter.com/mybinderteam",
+}
```

Comment:
Just like in the other PR, we can allow the edit option right?

```suggestion
html_theme_options = {
  "use_edit_page_button": True,
  "github_url": "https://github.com/jupyterhub/binderhub",
  "twitter_url": "https://twitter.com/mybinderteam",
}
```